resources/lib/bossanova808/playback.py

```
@@ -1,14 +1,17 @@
-from json import JSONDecodeError
+import json
 from dataclasses import dataclass
 from typing import List
 
-from xbmcgui import ListItem
-from infotagger.listitem import ListItemInfoTag
+import xbmc
+import xbmcgui
+import xbmcvfs
 
+from bossanova808.constants import *
```

Ruff (0.13.1)

`from bossanova808.constants import *` used; unable to detect undefined names

(F403)